stonesoup/base.py

```diff
@@ -159,8 +159,12 @@ def __init__(self, *args, **kwargs):
         cls._properties.update(
             (key, value) for key, value in namespace.items()
             if isinstance(value, Property))
-        # Optional arguments must follow mandatory
         for name in list(cls._properties):
+            # Remove items which are no longer properties
```

Comment:
Looks like these changes may be messing up the documentation. Under the documentation for `constant_derivative` property for the `RandomWalk` and `ConstantVelocity` classes I am getting the documentation for `Property`.

Reply:
Cheers, I don't seem to get this issue, could you possibly send a screenshot.